spec/models/email_log_spec.rb

```diff
@@ -110,4 +110,32 @@
       expect(EmailLog.find(email_log.id).bounce_key).to eq(hex)
     end
   end
+
+  describe "#cc" do
```

Comment:
This description feels odd to me. Usually `#` is used to indicate an instance method and we don't have a `#cc` instance method.